### PR TITLE
Renamed `sub-feed` to `service`

### PR DESCRIPTION
**Pull request by teizenman**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22432
https://github.com/demisto/etc/issues/22422

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [X] 5.5.0

## Does it break backward compatibility?
   - [X] Yes
       - Further details: All are required fields, and were renamed.
   - [ ] No

### CODE REVIEW

**Comment by roysagi** [comment]
Changed all the changelogs to the same line for consistency:
Renamed the **Sub-Feeds** parameter to **Services** in the instance configuration.

File: Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.yml [hunk 1]
```diff
@@ -84,9 +84,9 @@ configuration:
   required: true
   type: 4
 - defaultvalue: connectApi
-  display: Sub-Feeds
+  display: Services
   hidden: false
-  name: sub_feeds
+  name: x_services
```
Comment (ronykoz) [inline Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.yml]: ?

Comment (teizenman) [reply Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.yml]: Thought maybe `services` is a keyword, but it didnt help

**Comment by ronykoz** [comment]
Deleting this branch due to the new one

**Pull request closed without merging**